lib/instrumentation/undici.js

```diff
@@ -66,8 +66,8 @@ module.exports = function addUndiciChannels(agent, undici, modName, shim) {
    * We will check current segment for opaque and also attach
    * relevant headers to outgoing http request
    *
-   * @param {Object} params
-   * @param {Object} params.request undici request object
+   * @param {object} params
```

Comment:
just curious why this file is touched in the PR?

Reply:
https://github.com/newrelic/node-newrelic/pull/936#discussion_r723759706